### PR TITLE
Update the Servo dependency on realtime_tools

### PR DESCRIPTION
**Pull request by AndyZe**
Update the location of the `thread_priority` library which was previously lumped in with the `controller_manager` package

### CODE REVIEW

**Comment by tylerjw** [comment]
To get this to work I think you'll have to remove ros2_control from the repos file.

**Comment by tylerjw** [comment]
Per the CI failure on this one we are waiting on a Rolling sync for this to pass.

File: moveit2.repos [hunk 1]
```diff
@@ -1,21 +1,2 @@
 repositories:
-  moveit_msgs:
-    type: git
-    url: https://github.com/ros-planning/moveit_msgs.git
-    version: ros2
-  moveit_resources:
-    type: git
-    url: https://github.com/ros-planning/moveit_resources.git
-    version: ros2
```
Comment (rhaschke) [inline moveit2.repos:9]: As stated in https://github.com/ros-planning/moveit2/pull/1827, these MoveIt-related packages should be kept here as we typically release them together with MoveIt.

**Review by rhaschke — COMMENTED** [review]
Looks good, but keep `moveit_*` packages.

**Review by rhaschke — APPROVED** [review]

**Review by tylerjw — APPROVED** [review]

**Pull request merged**